### PR TITLE
added pd as namespace

### PR DESCRIPTION
**Pull request by dacoex**
- [x ] closes https://github.com/pydata/pandas/pull/12268#discussion_r55924987

This is the msg:
more consistent with code in the rest of the document

added more namespace

Generally, also numpy's randn should used as
np.random.randn

isolated namespace changes

Note:
I hope I got the rebase & merge right.

### CODE REVIEW

File: doc/source/io.rst [hunk 1]
```diff
@@ -2895,6 +2895,24 @@ everything in the sub-store and BELOW, so be *careful*.
 
 .. _io.hdf5-types:
 
+.. warning:: Hierarchical keys cannot be retrieved as dotted (attribute) access as described above for items stored under root node.
+
+    .. ipython:: python
```
Comment (jreback) [inline doc/source/io.rst]: pls take this out. This is a completly separate issue that we may or may not include.

File: doc/source/io.rst [hunk 2]
```diff
@@ -1475,7 +1475,7 @@ Writing to a file, with a date index and a date column
    dfj2['date'] = Timestamp('20130101')
```
Comment (sinhrks) [inline doc/source/io.rst]: Are these remainings are intentional? Pls change others and remove  ```from pandas import *```.

Comment (dacoex) [inline doc/source/io.rst]: Sorry bur this was not my change.
Something must habe gotten confused.

**Comment by jreback** [comment]
@dacoex no, @sinhrks is asking if you can make that change. should be straightforward.

**Comment by dacoex** [comment]
Ah, OK. will do.

**Comment by jreback** [comment]
pls build the docs. This makes most things actually fail as there are lots of additional imports that are needed (or ``pd.``) pre-pended

**Comment by jreback** [comment]
@dacoex can you update

**Comment by dacoex** [comment]
Sorry, I'd been busy with some other items off the coding domain.
What shall I update?
Reference:
https://github.com/pydata/pandas/pull/12608#issuecomment-204411671

File: doc/source/io.rst [hunk 3]
```diff
@@ -3127,7 +3126,7 @@ specified in the format: ``<float>(<unit>)``, where float may be signed (and fra
 .. ipython:: python
 
    from datetime import timedelta
-   dftd = DataFrame(dict(A = Timestamp('20130101'), B = [ Timestamp('20130101') + timedelta(days=i,seconds=10) for i in range(10) ]))
+   dftd = pd.DataFrame(dict(A = Timestamp('20130101'), B = [ Timestamp('20130101') + timedelta(days=i,seconds=10) for i in range(10) ]))
```
Comment (jreback) [inline doc/source/io.rst]: so since you eliminated the ``from pandas import *`` you need to build the docs and you will see failures, eg. ``pd.Timestamp``

Comment (dacoex) [inline doc/source/io.rst]: Ah. OK, thanks for the hint.

**Comment by jreback** [comment]
can you rebase and update

**Comment by jreback** [comment]
can you rebase / update?

**Pull request closed without merging**